static/js/src/base/navigation.ts

```diff
@@ -80,3 +80,5 @@ if (navAccountContainer) {
 }
 
 initNavDropdowns(".p-navigation__item--dropdown-toggle");
+
+export {};
```

Comment:
This seems like an antipattern to satisfy the automated check-gods. What's the actual error?

Reply:
hmm, well it's suggesting a blank export and honestly I don't think it's worth modifying the contents of the file too much for risk of breaking things so I'll :+1: this :shrug: